server/config/brain/chat/questions.en.json

```diff
@@ -1,12 +1,63 @@
 [
   {
     "label": "chat.greeting",
-    "questions": ["Hey!"],
-    "answers": ["Hey!"]
+    "questions": ["Hey!", "Hi", "Hello"],
+    "answers": ["Hey!", "Hi", "Hello, I'm Gladys."]
+  },
+  {
+    "label": "chat.greeting-gladys",
+    "questions": ["Hey Gladys !", "Hi Gladys", "Hello Gladys"],
+    "answers": [
+      "Hey, I'm your home automation assistant !",
+      "Hello, I am your home automation assistant !",
+      "Hi, I am your home automation assistant !"
+    ]
   },
   {
     "label": "chat.i-love-you",
     "questions": ["I love you"],
-    "answers": ["Me too!", "Of course you love me..."]
+    "answers": ["Me too!", "I knew that ..."]
+  },
+  {
+    "label": "chat.joke",
+    "questions": ["Gladys tell me a joke.", "Gladys do you know a joke ?", "Tell me a joke."],
+    "answers": [
+      "Do you know the devil's joke? Too bad it's hell !",
+      "The other day I told a joke about my clothes. They were folded !",
+      "Helvetica and Times New Roman walk into a bar “Get out of here!” shouts the bartender. “We don’t serve your type.",
+      "What do you say when calling a 4-headed monster? Hello, hello, hello, hello !",
+      "What do a postman and a knife thrower have in common? Both must have a lot of skill.",
+      "Why do we tell actors to “break a leg ? Because every play has a cast.",
+      "What’s the best thing about Switzerland ? I don’t know, but the flag is a big plus !",
+      "I invented a new word: Plagiarism !",
+      "How many times can you subtract 10 from 100 ? Once. The next time you would be subtracting 10 from 90.",
+      "Why Was Six Afraid of Seven ? Because seven ate nine !"
+    ]
+  },
+  {
+    "label": "chat.shifumi",
+    "questions": [
+      "Shall we make a shifumi ?",
+      "Gladys, are we playing shifumi ?",
+      "shifumi",
+      "shi fu mi",
+      "Gladys, are we doing a shi fu mi ?",
+      "Are we playing shifumi ?",
+      "Shall we make a stone, paper, scissors ?",
+      "Gladys, are we playing rock, paper, scissors  ?",
+      "Are we playing rock, paper, scissors ?"
+    ],
+    "answers": ["Stone !", "Paper !", "Scissors !"]
+  },
+  {
+    "label": "chat.heads-or-tails",
+    "questions": [
+      "Shall we do a coin toss ?",
+      "Gladys, are we playing a coin toss ?",
```

Comment:
Possible Correction: "Gladys, are we playing coin toss ?"

Reply:
ok I will add.